sale_automatic_workflow_payment_mode/data/automatic_workflow_data.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    sale_automatic_workflow_payment_mode for Odoo
+    Copyright (C) 2016 Sodexis
+    The licence is in the file __manifest__.py
+-->
+<odoo noupdate="1">
+
+        <record id="automatic_workflow_payment_filter" model="ir.filters">
+            <field name="name">Automatic Workflow Payment Filter</field>
+            <field name="model_id">account.invoice</field>
+            <field name="domain">[('state', '=', 'open')]</field>
+            <field name="user_id" ref="base.user_root"/>
+        </record>
+        <record id="sale_automatic_workflow.automatic_validation" model="sale.workflow.process">
+            <field name="register_payment" eval="0" />
+            <field name="payment_filter_id" ref="automatic_workflow_payment_filter" />
+        </record>
+        <record id="sale_automatic_workflow.manual_validation" model="sale.workflow.process">
```

Comment:
This xmlid is in a `noupdate="1"` section in `sale_automatic_workflow` so I think it is never updated here.